src/MountedTraversal.js

```diff
@@ -49,9 +49,8 @@ export function instHasClassName(inst, className) {
   if (!isDOMComponent(inst)) {
     return false;
   }
-  let classes = findDOMNode(inst).className || '';
-  classes = classes.replace(/\s/g, ' ');
-  return ` ${classes} `.indexOf(` ${className} `) > -1;
+  let classes = findDOMNode(inst).classList;
```

Comment:
:x: This will break in browsers that do not support `classList`, which includes IE 8 and 9, IE 10 and 11 for SVG (according to http://caniuse.com/#search=classlist), etc.

We simply must not use `classList` unconditionally - feature-detecting its support and falling back would be fine though.